src/Microsoft.ML.TorchSharp/NasBert/BaseModule.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using TorchSharp;
+
+namespace Microsoft.ML.TorchSharp.NasBert
```

Comment:
>Microsoft.ML.TorchSharp.NasBert [](http://example.com/codeflow?start=10&length=31)

Is it the convention to use terms like `NasBert` in the namespace? 

Would it make sense more to have the base calls `BaseModule` named like `NasBertModule` and keep the namespace as `Microsoft.ML.TorchSharp`

Reply:
I put it in 'NasBert` namespace because its only the BaseModule for the current implementation we are using from MSR. Other models for TorchSharp may not use this at all.